app/helpers/expenses_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ExpensesHelper
+  def sum_amounts(expenses)
```

Comment:
- [ ] If an input filed is empty, it gives the error; 
![image](https://user-images.githubusercontent.com/58504655/227233860-20b9716f-fffe-4e80-8631-8677c97792f2.png)